types/mutatingwh_config.go

```diff
@@ -0,0 +1,95 @@
+package types
+
+import (
+	"strings"
+
+	"github.com/koki/json"
+	serrors "github.com/koki/structurederrors"
+	"k8s.io/api/admissionregistration/v1beta1"
+)
+
+type MutatingWebhookConfigWrapper struct {
+	MutatingWebhookConfig MutatingWebhookConfig `json:"mutatingwh_config"`
+}
+
+type MutatingWebhookConfig struct {
+	Version     string            `json:"version,omitempty"`
+	Cluster     string            `json:"cluster,omitempty"`
+	Name        string            `json:"name,omitempty"`
+	Namespace   string            `json:"namespace,omitempty"`
+	Labels      map[string]string `json:"labels,omitempty"`
+	Annotations map[string]string `json:"annotations,omitempty"`
+
+	Rules map[string][]MutatingWebhookRuleWithOperations `json:"rules,omitempty"`
+}
+
+type MutatingWebhookRule struct {
```

Comment:
Is this used anywhere?

Reply:
I see where this is being used. 

I would rather use one struct, and add the `omitempty` tag to the Operations Slice.